test/csuite/wt2999_join_extractor/main.c

```diff
@@ -28,15 +28,17 @@
 #include "test_util.h"
 
 /*
- * JIRA ticket reference: WT-2999
+ * custom_extract1 --
+ *     JIRA ticket reference: WT-2999
  *
  * Test case description: Create a table that stores ~4K size blobs; two indices are defined using a
- * pair of custom extractors that pull the first and second 32-bit integers from the blob. A simple
- * join is created using the two indices, and iterated.
+ *     pair of custom extractors that pull the first and second 32-bit integers from the blob. A
+ *     simple join is created using the two indices, and iterated.
  *
  * Failure mode: When a custom extractor is used with cursor joins, there are memory leaks at the
- * point where the extractor sets the key.
+ *     point where the extractor sets the key.
  */
+
 /*
```

Comment:
Need to be removed

Reply:
Realised that the comment didn't describe the function but the whole test! Fixed it up